src/main/java/sx/blah/discord/handle/obj/Verification.java

```diff
@@ -0,0 +1,45 @@
+package sx.blah.discord.handle.obj;
+
+/**
+ * Represents a verification level for a guild
+ */
+public enum Verification {
+	/**
+	 * Represents a verification level of None
+	 */
+	NONE,
+	/**
+	 * Represents a verification level of Low
+	 */
+	LOW,
+	/**
+	 * Represents a verification level of Medium
+	 */
+	MEDIUM,
+	/**
+	 * Represents a verification level of (╯°□°）╯︵ ┻━┻
+	 */
+	HIGH;
+
+	/**
+	 * Gets the Verification object for an integer level, or null if outside 0-3
+	 *
+	 * @param level The verification level as an int.
+	 * @return The Verification object for the given level.
+	 */
+	public static Verification getByLevel(int level) {
+		try {
+			return Verification.values()[level];
+		} catch (ArrayIndexOutOfBoundsException e) {
```

Comment:
Now that I think about it, it's not that bad. The only tiny tiny thing I see is that `e` could be renamed to `ignored` (to prevent IntelliJ from marking that as weird).